pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1812,90 +1812,12 @@ func TestTaintPressureNodeDaemonLaunchesPod(t *testing.T) {
 	}
 }
 
-// When ScheduleDaemonSetPods is disabled, DaemonSet should launch a critical pod even when the node has insufficient free resource.
-func TestInsufficientCapacityNodeDaemonLaunchesCriticalPod(t *testing.T) {
```

Comment:
I still don't understand why these tests are being removed.  Is this functionality not applicable when using the equivalent PriorityClass?  Is this functionality proven by other tests instead?

Reply:
addressed